ql/src/test/queries/clientpositive/udf_array_remove.q

```diff
@@ -0,0 +1,40 @@
+--! qt:dataset:src
+
+-- SORT_QUERY_RESULTS
+
+set hive.fetch.task.conversion=more;
+
+DESCRIBE FUNCTION array_remove;
+DESCRIBE FUNCTION EXTENDED array_remove;
+
+-- evaluates function for array of primitives
```

Comment:
It could be nice if we have a test case where the second argument is NULL.

Reply:
`int type element is expected at function array_remove(array<int>,int), but void is found` is thrown if we have null in second argument. so added that test case in Junit file <`runAndVerify(asList(i1, i2, i3, i4),null,null);`>

Reply:
Thanks. Just for your info, I guess `array_remove({list}, CAST(null AS {element type})` works.